tests/std/tests/P1522R1_difference_type/test.cpp

```diff
@@ -559,6 +711,13 @@ constexpr bool test_cross() {
 
     TEST(_Uint128{1} << _Int128{43}, _Uint128{1ull << 43});
     TEST(_Int128{1} << _Uint128{43}, _Int128{1ull << 43});
+    TEST(_Uint128{-1} << _Int128{43}, _Uint128(0ull - (1ull << 43), ~0ull));
+    TEST(_Int128{-1} << _Uint128{43}, _Int128(0ull - (1ull << 43), ~0ull));
+
+    TEST(_Uint128(0, 1) >> _Int128{43}, _Uint128(1ull << 21));
+    TEST(_Int128(0, 1) >> _Uint128{43}, _Int128(1ull << 21));
+    TEST(_Uint128(0, ~0ull) >> _Int128{43}, _Uint128(~((1ull << 21) - 1), (1ull << 21) - 1));
+    TEST(_Int128(0, ~0ull) >> _Uint128{43}, _Int128(~((1ull << 21) - 1), ~0ull));

```

Comment:
Braces were used everywhere else to construct `_Int128` and `_Uint128` temporaries, but this is now using some parens.

Reply:
Yeah, I'll `TEST(())` instead to guard the commas. That way, the anomaly is right next to a macro name which should make it immediately obvious why there's an anomaly.